README.md

```diff
@@ -1,9 +1,78 @@
 # Generic Low Overhead Message Exchange (GLOME)
 
-**This is not an officially supported Google product.**
+**GLOME Login** is a [challenge-response authentication mechanism](https://en.wikipedia.org/wiki/Challenge%E2%80%93response_authentication). It resembles [one-time authorization codes](https://en.wikipedia.org/wiki/One-time_password) (aka OTPs) but is different from [HOTP] and [TOTP] in the following ways:
+
+- It is stateless (unlike [HOTP]).
+- It does not depend on time (unlike [TOTP]).
+- It does not require a predefined secret sharing (unlike [HOTP] and [TOTP]).
+
+These properties make it a good choice for low dependency environments (e.g., devices with no persistent storage, source of entropy, or a real-time clock). It can be also useful for managing access to a large fleet of hosts where synchronising state or sharing a predefined secrets can be a challenge. 
```

Comment:
Technically poor entropy is still a problem, no? You are technically vulnerable to KCI in that case? I.e. you know the server's public key and if you can guess the ephemeral private key then you can compute the shared secret. Of course you'll only have one try - and I think there's nothing in the challenge that'd show what would be valid...